src/modules/Calculator.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+export default class Calcuclator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <main className="calculator">
+        <div className="flex-center result">0</div>
+        <div className="calculator-btns">
+          <button type="button">AC</button>
+          <button type="button">+/-</button>
+          <button type="button">%</button>
+          <button type="button" className="backgroundOrange">/</button>
```

Comment:
- [ ] Kindly use the correct division sign in the design for your styling as well. If you do not have the sign on your keyboard, consider using HTML entity code.

  ![image](https://user-images.githubusercontent.com/81468675/211346036-71de4fc2-3425-432d-98e2-81fabfaa4cf7.png)